src/test/scala/uk/gov/ons/registers/stepdefs/CommonSteps.scala

```diff
@@ -1,14 +1,16 @@
 package uk.gov.ons.registers.stepdefs
 
+import scala.collection.JavaConversions._
```

Comment:
when i was going through the checkstyle warnings when i first started i had to go through and change javaconversions to javaconverters as it said javaconversions is deprecated. not sure if it would affect this but thought worth a mention.

Reply:
I vaguely remember this. Good recall... I shall investigate.